omod/src/main/resources/config.xml

```diff
@@ -265,6 +265,14 @@
         </description>
     </globalProperty>
 
+    <globalProperty>
+        <property>coreapps.conditionListCriteria</property>
```

Comment:
Of coreapps.conditionListCriteria and coreapps.conditionListClasses, which is clearer?

Reply:
Both seem to be fine. Do you suggest to change it to coreapps.conditionListClasses ?

Reply:
Are you sure one is not closer to the actual values supplied?

Reply:
I will change the property name to coreapps.conditionListClasses